api/room/schema.py

```diff
@@ -137,6 +168,9 @@ def resolve_rooms(self, info, **kwargs):
 
 
 class UpdateRoom(graphene.Mutation):
+    """
+        Update a givem room
+    """
     class Arguments:
```

Comment:
You can also document this like in the create method, The idea is I can go to any mutation and know what type of data I should send and what I should input as the parameters of my mutation.

Reply:
I thought it would be repetition as the arguments are already listed as in the screenshot below. What do you think?
![image](https://user-images.githubusercontent.com/29709981/53576377-8daa9780-3b84-11e9-8138-b19d652858b4.png)

Reply:
It is a valid point but try to imagine I just went to the UpdateTag endpoint directly without looking at the CreateTag.